requirements.txt

```diff
@@ -11,6 +12,7 @@ django-model-utils==3.1.2
 django-munin==0.2.1
 python-memcached==1.59
 lxml==4.2.5
+libxml2-python3==2.9.5
```

Comment:
tu es sûr de ça?

Reply:
Quand on fait apt-get ça n'installe pas l'une des dernières versions ?

Reply:
requirements.txt est là pour donner les dépendances python, pas pour les dépendances apt.

Reply:
Les deux sont en python et sont sur pip : 

 - https://pypi.org/project/setuptools/
 - https://pypi.org/project/libxml2-python3/

Reply:
mais pourquoi as-tu besoin de les ajouter?

Reply:
Je les ai déplacé, au lieu d'utiliser apt-get, j'utilise pip pour pouvoir renseigner un numéro de version

Reply:
bah on n'a jamais installé setuptools avec apt-get

pour libxml2-python3, why not même si j'ai des doutes

Reply:
@artragis Pourtant ils sont bien présent dans le script actuel : https://github.com/zestedesavoir/zds-site/blob/dev/scripts/install_zds.sh#L45